v3/operations.go

```diff
@@ -12393,6 +12393,13 @@ func (c Client) GetSKSClusterInspection(ctx context.Context, id UUID) (*GetSKSCl
 	return bodyresp, nil
 }
 
+type CreateSKSNodepoolRequestPublicIPAssignment string
+
+const (
+	CreateSKSNodepoolRequestPublicIPAssignmentInet4 CreateSKSNodepoolRequestPublicIPAssignment = "inet4"
+	CreateSKSNodepoolRequestPublicIPAssignmentDual  CreateSKSNodepoolRequestPublicIPAssignment = "dual"
+)
```

Comment:
@pierre-emmanuelJ I don't like that we are generating two sets of constants with a different type for the same enum.(not blaming anyone, just noticing this now) You can see what I mean by comparing this section to [this one](https://github.com/exoscale/egoscale/pull/649/files#diff-5babdd0ca7b79139b430ed1d34e44ceacba9e02e56e941e43a936afaf669d5b5R12572-R12577)

The problem is that the public-api spec contains the enum [multiple times](https://github.com/exoscale/egoscale/pull/649/files#diff-3dcdfbe89405309063054daf48a093a6dfb47bbabe2112137aa6c2176aa34265R4668-R13776). In my opinion we could consider two things:
1. change the spec so that it just generates one enum and reuses it in both update and create calls.
2. extend the egoscale v3 generator to create enums instead of consts. (If this is backwards incompatible change let's not do it.)

These suggestions are cosmetic. The PR looks good to me though and I'm able to implement the [feature in tf](https://github.com/exoscale/terraform-provider-exoscale/pull/379/files#diff-a7712fedde68fe07a37009cb84143bdee4ab3668aab77916591ff2555815e13fR401). So it's not a big problem.

Feel free to merge if you think it's all right or let me know if we should change something.

Reply:
Thanks, let's discuss it internally to take an appropriate decision :)